src/ert/scheduler/job.py

```diff
@@ -57,7 +61,9 @@ def __init__(self, scheduler: Scheduler, real: Realization) -> None:
         self.started = asyncio.Event()
         self.returncode: asyncio.Future[int] = asyncio.Future()
         self.aborted = asyncio.Event()
-        self._scheduler = scheduler
+        self._scheduler: Scheduler = scheduler
+        self._callback_status_msg: str = ""
+        self._max_submit: int = 2
```

Comment:
Why is `_max_submit` set as an attribute here? I think this one was suppose to just come as function parameter for `__call__`?